### PR TITLE
Add listener test case and fix module version in listener

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

**Comment by ThishaniLucas** [comment]
> Aren't we running these tests per PR? Noticed tests are skipped in Github actions.

Yes. We need to set up a dedicated Salesforce instance for the tests. Also, the tests fail intermittently due to delays with the bulk operations. ATM, the tests are run locally. Will enable them once these are addressed.

File: sfdc-connector/src/sfdc/tests/listener_test.bal [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import ballerina/test;
```
Comment (daneshk) [inline sfdc-connector/src/sfdc/tests/listener_test.bal:1]: Add license header

**Comment by niveathika** [comment]
@ThishaniLucas Ack

**Comment by daneshk** [comment]
> > Aren't we running these tests per PR? Noticed tests are skipped in Github actions.
> 
> Yes. We need to set up a dedicated Salesforce instance for the tests. Also, the tests fail intermittently due to delays with the bulk operations. ATM, the tests are run locally. Will enable them once these are addressed.

We need to enable the tests soon. Let's figure out how we can do it

**Comment by ThishaniLucas** [comment]
> We need to enable the tests soon. Let's figure out how we can do it

Ack. Created an issue [1] to keep note of this.

[1] https://github.com/ballerina-platform/module-ballerinax-sfdc/issues/110

**Pull request merged**